core/ts.core/src/ts/utils/ZipUtils.java

```diff
@@ -172,12 +173,22 @@ public static void extractTar(File file, File destination) throws IOException {
 					break;
 				case TarEntry.LINK:
 					File linkFile = new File(destination, outFilename);
+					// Be sure that parent file exists
+					File linkBaseDir = linkFile.getParentFile();
+					if (!linkBaseDir.exists()) {
+						linkBaseDir.mkdirs();
+					}
 					Path target = new File(linkFile.getParentFile(), entry.getLinkName()).toPath();
```

Comment:
Ummm...  seems that I can't "really" create a tar.gz file that contains "hard links" cause when packaged, the "hard link" is a copy of the original linked file (so, no information that it was a link).  Any way I'm going to modify that part of the code (but I think that is never been executed).